### PR TITLE
hpc: Add pdsh sanity check

### PR DESCRIPTION
**Pull request by Soulofdestiny**

### CODE REVIEW

**Comment by Soulofdestiny** [comment]
master: http://drnek.arch.suse.de/tests/778
slave: http://drnek.arch.suse.de/tests/779

poo: https://progress.opensuse.org/issues/15520

File: tests/hpc/pdsh_slave.pm [hunk 1]
```diff
@@ -0,0 +1,63 @@
+# SUSE's openQA tests
+#
+# Copyright © 2009-2013 Bernhard M. Wiedemann
```
Comment (asmorodskyi) [inline tests/hpc/pdsh_slave.pm]: please remove this part

File: tests/hpc/pdsh_master.pm [hunk 2]
```diff
@@ -0,0 +1,65 @@
+# SUSE's openQA tests
+#
+# Copyright © 2009-2013 Bernhard M. Wiedemann
```
Comment (asmorodskyi) [inline tests/hpc/pdsh_master.pm]: please remove this part

**Comment by asmorodskyi** [comment]
generally LGTM , just minor license issue , please remove it and I will merge it

**Comment by Soulofdestiny** [comment]
done

**Pull request merged**